blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -35,7 +48,7 @@ export const GalleryPresentation = ({
 		AdBlock = () => <p>Ad block not found</p>;
 	}
 
-	const { resizerURL, galleryCubeClicks, locale = "en" } = getProperties(arcSite);
+	const { galleryCubeClicks, locale } = getProperties(arcSite);
```

Comment:
no default?  This will cause issues won't it?
```suggestion
	const { galleryCubeClicks, locale = "en" } = getProperties(arcSite);
```

Reply:
locale is en by default

Reply:
I thought it was confirmed that it wasn't because fusion didn't want to default it.  If that's the case, I'm happy about that... but I had heard different.  @matthewroach: do you know?

Reply:
We can assume it's going to be passed as `en` as at minimum the default value in properties should be set via themes settings

Reply:
so we can start leaving this off?